src/modules/aws.rs

```diff
@@ -44,42 +44,34 @@ fn get_aws_region_from_config(aws_profile: &Option<String>) -> Option<String> {
     Some(region.to_string())
 }
 
-fn get_aws_region() -> Option<(String, String)> {
-    env::var("AWS_DEFAULT_REGION")
-        .ok()
-        .map(|region| (String::new(), region))
-        .or_else(|| {
-            let aws_profile = env::var("AWS_PROFILE").ok();
-            let aws_region = get_aws_region_from_config(&aws_profile);
-
-            if aws_profile.is_none() && aws_region.is_none() {
-                None
-            } else {
-                Some((
-                    aws_profile.unwrap_or_default(),
-                    aws_region.unwrap_or_default(),
-                ))
-            }
-        })
-        .or_else(|| {
-            env::var("AWS_REGION")
-                .ok()
-                .map(|region| (String::new(), region))
-        })
+fn get_aws_region() -> (String, String) {
+    match (
+        env::var("AWS_PROFILE").ok(),
```

Comment:
Not a big deal, but right now it looks like we get environment variables that may or may not be used.
Would it be possible to structure it in a way that we check `config.displayed_items` before retrieving the necessary environment variables?

Reply:
Good idea. I tried to do just that.

I still do an unnecessarily call to AWS_PROFILE if AWS_DEFAULT_PROFILE is set. But it meant a bigger refacto.